lab-taylor/route/order-route.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const Order = require('../model/order');
+const response = require('../lib/response');
+
+const Storage = require('../lib/storage');
+const storage = new Storage(`${__dirname}/../data`);
+
+module.exports = function(router) {
+  router.post('/api/order', function (req,res) {
+    const order = new Order(req.body.item, req.body.qty);
+
+    if (!req.body || !req.body.item || !req.body.qty) {
+      return response(400, 'bad request')(res);
+    }
+
+    storage.setOrder('order', order)
+    .then(function(order) {
+      return response(200, order)(res);
+    }).catch (function () {
+      return response(400, 'bad request')(res);
+    });
+  })
+  .get('/api/order', function(req, res){
+    if (!req.url.query.id) return response(400, 'bad request')(res);
+
+    storage.fetchOrder('order', req.url.query.id)
+    .then(function (order) {
+      return response(200, order)(res);
+    }).catch(function () {
+      return response(404, 'not found')(res);
+    });
+  })
+  .put('/api/order', function(req, res){
```

Comment:
This wasn't required, but excellent job!  I would suggest instead of writing the update logic here, it may be something good to add to the storage module instead.